### PR TITLE
Ai 19 invites and teamless users

### PR DESCRIPTION
**Pull request by angouleme**
[jira card](https://netguru.atlassian.net/browse/AI-19)

What it does:
* lists users with 0 teams on GitHub after the diff is calculated
* allows removing from organisation users who are both not assigned to team on github and missing from dataguru
* refactors listing users that don't have 2fa enabled on github (so listing both no 2fa members & teamless members would use same mechanism and classes)

### CODE REVIEW

**Comment by mczuchnowski** [comment]
:+1: make it happen

File: Gemfile [hunk 1]
```diff
@@ -8,7 +8,7 @@ gem 'celluloid'
 gem 'data_guru'
 gem 'decent_exposure'
 gem 'font-awesome-rails'
-gem 'github_api'
+gem 'github_api', '0.13.0'
```
Comment (lubieniebieski) [inline Gemfile]: why do we lock this? no info:/

File: app/models/github_integration/actions/list_users.rb [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+          dg_user = dg_users.find do |user|
+            user.github.to_s.casecmp(gh_member['login']).zero?
+          end
+          build_user(dg_user, gh_member)
+        end
+      end
+
+      def build_user(dg_user, gh_member)
+        if dg_user.nil?
+          DataGuruNilUser.new(gh_member)
```
Comment (lubieniebieski) [inline app/models/github_integration/actions/list_users.rb]: NullObject pattern - :+1:

File: Gemfile (hunk 1 above)
Comment (angouleme) [inline Gemfile]: This release brings new features (although we're not using them yet) and doesn't break anything else (tested). Would changing it to `~> 0.13.0` make more sense?

Comment (lubieniebieski) [inline Gemfile]: let's just update it without specifying a version

Comment (angouleme) [inline Gemfile]: Or, since I actually didn't use features from new release, we could stay with the current version and bump it when it's really needed.

**Pull request merged**